src/libs/Middleware/RecheckConnection.js

```diff
@@ -1,32 +1,20 @@
 import CONST from '../../CONST';
-import NetworkConnection from '../NetworkConnection';
-
-/**
- * @returns {Function} cancel timer
- */
-function startRecheckTimeoutTimer() {
-    // If request is still in processing after this time, we might be offline
```

Comment:
I don't think this was necessary because the timeout for an internet reachability test is [built-in to NetInfo](https://github.com/react-native-netinfo/react-native-netinfo/blob/83c1e0d9573f8e1a439f2e6b58f689739cdfdad3/src/internal/internetReachability.ts#L70-L144) (see `reachabilityRequestTimeout`)